tests/real/community.rs

```diff
@@ -62,6 +62,77 @@ fn roundtrip() {
     assert_eq!(job.args(), &[Value::from("z")]);
 }
 
+#[test]
+#[cfg(feature = "tls")]
+fn roundtrip_tls() {
+    use native_tls::{Certificate, TlsConnector};
+    use std::{env, fs::File, io::Read};
+
+    if env::var_os("FAKTORY_URL").is_none() || env::var_os("FAKTORY_URL_SECURE").is_none() {
+        return;
+    }
+    let local = "roundtrip_tls";
+
+    let (tx, rx) = sync::mpsc::channel();
+    let tx = sync::Arc::new(sync::Mutex::new(tx));
+    let mut c = ConsumerBuilder::default();
+    c.hostname("tester".to_string()).wid(local.to_string());
+    {
+        let tx = sync::Arc::clone(&tx);
+        c.register(local, move |j| -> io::Result<()> {
+            tx.lock().unwrap().send(j).unwrap();
+            Ok(())
+        });
+    }
+
+    let mut cert = String::new();
+    let cert_path = env::current_dir()
+        .unwrap()
+        .join("docker")
+        .join("certs")
+        .join("faktory.local.crt");
+    File::open(cert_path)
+        .and_then(|mut f| f.read_to_string(&mut cert))
+        .unwrap();
+    let tls = || {
+        let connector = if cfg!(target_os = "macos") {
+            TlsConnector::builder()
+                // Danger! Only for testing!
+                // On the macos CI runner, the certs are not trusted:
+                // { code: -67843, message: "The certificate was not trusted." }
+                .danger_accept_invalid_certs(true)
+                .build()
+                .unwrap()
```

Comment:
Made tests on macos [succeed](https://github.com/jonhoo/faktory-rs/actions/runs/7773140130/job/21196438373). Although the tests are  no longer executed on a macos runner, leaving this dangerous workaround behind for possible future use.  ...and as a reminder of how painful that was :)

Reply:
Given this is all testing in CI anyway, I'd also be fine just running the entire test (on all OSes) with `danger_accept_invalid_certs`. But leaving it here with docs like you have seems fine :+1: